taiga2/controllers/models_controller.py

```diff
@@ -1249,7 +1251,9 @@ def add_s3_datafile(
     return new_datafile
 
 
-def _add_virtual_datafile(name, datafile_id):
+def _add_virtual_datafile(
+    name, datafile_id, custom_metadata: Optional[Dict[str, Any]] = None
```

Comment:
This is suggesting that metadata values can be anything. I was thinking to keep things simple, values would always be strings. 

However, Taiga is an internal tool. So, sure why not. :) Sounds fine to me.